cosa/analyzers/mcsolver.py

```diff
@@ -234,7 +234,7 @@ def _write_smt2_comment(self, solver, line):
 
     def _formula_to_smt2(self, formula):
         buf = cStringIO()
-        printer = SmtDagPrinter(buf)
+        printer = SmtPrinter(buf)
```

Comment:
Please add a comment why switching from SmtDagPrinter to SmtPrinter here.

Reply:
Good catch, thanks! That was actually just a debug change -- my bad. I thought I changed this already but I might have force pushed so maybe github didn't update? Anyway, it should be fixed now.